loadbalancer.tf

```diff
@@ -0,0 +1,47 @@
+################################################################################
```

Comment:
nice to have it in a separate file, 
maybe we should do a stand alone module for it and import it here